alma/controllers/front/ipn.php

```diff
@@ -68,13 +68,12 @@ public function postProcess()
         $validator = new PaymentValidation($this->context, $this->module);
 
         try {
-            Logger::instance()->debug('payment_validate');
             $validator->validatePayment($paymentId);
         } catch (PaymentValidationError $e) {
-            Logger::instance()->error('payment_validation_error - Message : ' . $e->getMessage());
+            Logger::instance()->error('ipn payment_validation_error - Message : ' . $e->getMessage());
             $this->fail($e->getMessage());
         } catch (Exception $e) {
-            Logger::instance()->error('payment_error - Message : ' . $e->getMessage());
+            Logger::instance()->error('ipn payment_error - Message : ' . $e->getMessage());
             $this->fail($e->getMessage());
         }
 
```

Comment:
I see only "PaymentValidationError" exception . Are you sure  y ou have "* @throws PaymentValidationError|PrestaShopException|Exception"  ?

Reply:
@Francois-Gomis 
I edited the Exception about our speaking.
I let you check if is good :)